go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/buildpacks/pack v0.14.0
+	github.com/docker/docker v1.13.1
```

Comment:
I don't see it used anywhere, just be aware that the version will be overridden by `replace` block below. In addition I was under impression that `v1.13.1` doesn't work with current buildpack version. 

Dependencies, dependencies, ... :)

Reply:
Good catch, thanks! This is a leftover from previous iteration.  Fixed